website/styles/components/_example.scss

```diff
@@ -29,4 +29,82 @@
 
 .example--truncate > * {
   max-width: rebase(15rem);
+}
+
+.example__inline {
+  display: flex;
+  gap: rebase(0.5rem);
+  align-items: center;
+}
+
+//Special style for the example of Modal
+.ex-modal {
+  &__header {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+  }
+
+  &__profile {
+    display: flex;
+    align-items: center;
+    gap: rebase(1rem);
```

Comment:
This is more of a question but what is rebase? I haven't seen this before.

Reply:
The rebase is a helper which transforms the values into base 10. 
The component is developed with tokens on base10 and In the doc we base ourselves on a base 8.

**Example**
Component: 1rem = 10px;
Doc: 1rem = 16px;